src/server/mcp.ts

```diff
@@ -290,12 +282,22 @@ export class McpServer {
     }
 
     const field = prompt.argsSchema.shape[request.params.argument.name];
-    if (!(field instanceof Completable)) {
+    const defLike = (field as unknown as { _def?: { typeName?: unknown } })
+      ._def;
+    if (!defLike || defLike.typeName !== McpZodTypeKind.Completable) {
```

Comment:
again, not sure why `as unknown as` is required, I also suggest to use a helper function, something like 
```suggestion
    if (notCompletable(field)) {
```
could look nicer